types/polygon/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "member-access": false
```

Comment:
Why is this needed? Is it possible to remove it?

Reply:
I added this, to be able to write `public` on functions and properties, without `public' is implicit` warning.
Now i removed `public` and the rule. Hope this is ok.